src/components/Home.js

```diff
@@ -0,0 +1,23 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { getData } from '../redux/home/home';
+import Card from './Card';
+
+export default function Home() {
```

Comment:
- [ ] Kindly implement a **redux filter/ search** functionality on your Homepage so as to enable filtering of countries.